openweather.js

```diff
@@ -131,6 +133,16 @@ function onPositionUpdate(value, callback) {
 
         owm.setCoordinate(value.latitude, value.longitude);
         log("OWM Coordinates "+value.latitude+","+value.longitude);
+
+        // get the Temperature  
+	    owm.getTemperature(function(err, temp){
+		    if (!err)
+            {
+                console.log('Current temperature:'+ temp);
```

Comment:
plugin should not write directly to the console - use log function instead.

Reply:
fixed